msfr/steady/msfr_neutronics.i

```diff
@@ -47,11 +48,32 @@
 []
 
 [AuxVariables]
+  [v_x]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [v_y]
+    order = CONSTANT
+    family = MONOMIAL
+  []
```

Comment:
what is this doing here

Reply:
shouldn't be there. Initially it was there to see if I could save some simulation time through transfers.
I will delete it